sycl/test/check_device_code/matrix/matrix_load_store_as.cpp

```diff
@@ -9,55 +9,32 @@
 using namespace sycl;
 using namespace sycl::ext::oneapi::experimental::matrix;
 
-int main(void) {
-  queue q;
-  unsigned short *A = malloc_shared<unsigned short>(8 * 16, q);
-  unsigned short *B = malloc_shared<unsigned short>(16 * 16, q);
-  float *C = malloc_shared<float>(8 * 16, q);
-
-  auto pA = multi_ptr<unsigned short, access::address_space::global_space>(A);
-  auto pB = multi_ptr<unsigned short, access::address_space::global_space>(B);
-  auto pC = multi_ptr<float, access::address_space::global_space>(C);
-
-  q.submit([&](handler &h) {
-    local_accessor<unsigned short, 2> tileA{{8, 16}, h};
-
-    h.parallel_for(
-        nd_range<2>({1, 16}, {1, 16}),
-        [=](nd_item<2> it) [[intel::reqd_sub_group_size(16)]] {
-          joint_matrix<sub_group, unsigned short, use::a, 8, 16,
-                       layout::row_major>
-              tA;
-          joint_matrix<sub_group, unsigned short, use::b, 16, 16,
-                       layout::ext_intel_packed>
-              tB;
-          joint_matrix<sub_group, float, use::accumulator, 8, 16> tC;
-
-          sub_group sg = it.get_sub_group();
-          vec<unsigned short, 8> slmvec = sg.load<8>(pA);
-          sg.store<8>(
-              tileA.template get_multi_ptr<sycl::access::decorated::yes>(),
+SYCL_EXTERNAL [[intel::reqd_sub_group_size(16)]] void matrix_store_as(
```

Comment:
Matrix tests belong to a different team (or at least should) and thus should go to a separate PR.

Reply:
According to the CODEOWNERS file, this is _technically_ ours, although that is probably a typo. Either way, I've reverted the changes, and will be making a new PR for the matrix team.

Should I do anything about CODEOWNERS? I will communicate with the matrix team regardless.

Reply:
> Should I do anything about CODEOWNERS? I will communicate with the matrix team regardless.

Yes, please create a PR to update it.